src/Aspire.Dashboard/Extensions/FluentUIExtensions.cs

```diff
@@ -25,4 +25,29 @@ internal static class FluentUIExtensions
 
         return attributes;
     }
+
+    public static Dictionary<string, object> GetOpenTextVisualizerAdditionalAttributes(string textValue, string textValueDescription, params (string Attribute, object Value)[] additionalAttributes)
+    {
+        var attributes = new Dictionary<string, object>(AttributeKeyComparer.Instance)
+        {
+            { "data-text", textValue },
+            { "data-textvisualizer-description", textValueDescription }
+        };
+
+        foreach (var (attribute, value) in additionalAttributes)
+        {
+            attributes.Add(attribute, value);
+        }
+
+        return attributes;
+    }
+
+    private class AttributeKeyComparer : IEqualityComparer<string>
```

Comment:
Why is this needed? Could https://learn.microsoft.com/en-us/dotnet/api/system.stringcomparer.ordinal?view=net-8.0 be used?

Also, it could be added as another property to StringComparers and StringComparisons.

Reply:
Drew suggested it, I like using `StringComparer.Ordinal`. Changed